app/javascript/packs/application/App.js

```diff
@@ -113,6 +122,11 @@ class App extends Component {
               onCreateTour={this.registerTour}
               currentStaffUser={this.state.currentStaffUser}
             />
+            <TourContainer
+              path="/tours/:tourId"
+              tour={this.state.tourView.data}
+              showTour={this.showTour}
```

Comment:
I think the prop name here should be consistent with the `api` function that it calls, which is `getTour`